dolphinscheduler-server/pom.xml

```diff
@@ -143,6 +162,12 @@
             <groupId>org.springframework</groupId>
             <artifactId>spring-test</artifactId>
         </dependency>
+
+        <dependency>
+            <groupId>org.ohdsi.sql</groupId>
+            <artifactId>SqlRender</artifactId>
+            <version>1.7.0-SNAPSHOT</version>
```

Comment:
Very cool, but I noticed that you have introduced a new license. License is a more important issue. You can follow this document to proceed https://dolphinscheduler.apache.org/en-us/community/development/DS-License .html review.

btw, don't use the snap package